config/dev.exs

```diff
@@ -62,11 +62,10 @@ config :code_corps, CodeCorps.Mailer,
 config :code_corps,
   postmark_receipt_template: "123"
 
-# If the dev environment has no CLOUDEX_API_KEY set, we want the app to still run,
-# with cloudex in test API mode
+# If the dev environment has no CLOUDEX_API_KEY set, we want the app
+# to still run, with cloudex in test API mode
 if System.get_env("CLOUDEX_API_KEY") == nil do
-  config :cloudex, :cloudinary_api, Cloudex.CloudinaryApi.Test
```

Comment:
This is actually a documented `Cloudex` approach of enabling test mode for the API. Don't have a link right now, but should probably look into the Clodex docs to see what it does. Not sure if we wanna remove it.